src/main/flight/pid.h

```diff
@@ -188,11 +188,11 @@ typedef struct pidProfile_s {
     uint8_t ff_boost;                       // amount of high-pass filtered FF to add to FF, 100 means 100% added
     char profileName[MAX_PROFILE_NAME_LENGTH + 1]; // Descriptive name for profile
 
-    uint8_t idle_min_rpm;                   // minimum motor speed enforced by integrating p controller
-    uint8_t idle_adjustment_speed;          // how quickly the integrating p controller tries to correct
-    uint8_t idle_p;                         // kP
-    uint8_t idle_pid_limit;                 // max P
-    uint8_t idle_max_increase;              // max integrated correction
+    uint8_t idle_min_rpm;               // minimum motor speed enforced by the dynamic idle controller
```

Comment:
These should be renamed back to their original names as well I think.

Reply:
`idle_p` wasn't actually what it is named as, it's actually the integral sum kI.
the new code doesn't use a customisable `idle_pid_limit`
naming them consistently for the application they are now used for is appropriate, I think.

Reply:
It's looking likely that a single gain adjustment may be sufficient to modify P and I gain at the same time.